src/messages/Message.cpp

```diff
@@ -26,45 +19,57 @@ Message::~Message()
     DebugCount::decrease("messages");
 }
 
-SBHighlight Message::getScrollBarHighlight() const
+ScrollbarHighlight Message::getScrollBarHighlight() const
 {
     if (this->flags.has(MessageFlag::Highlighted) ||
         this->flags.has(MessageFlag::HighlightedWhisper))
     {
-        return SBHighlight(this->highlightColor);
+        return {
+            this->highlightColor,
+        };
     }
-    else if (this->flags.has(MessageFlag::Subscription) &&
-             getSettings()->enableSubHighlight)
+
+    if (this->flags.has(MessageFlag::Subscription) &&
+        getSettings()->enableSubHighlight)
     {
-        return SBHighlight(
-            ColorProvider::instance().color(ColorType::Subscription));
+        return {
+            ColorProvider::instance().color(ColorType::Subscription),
+        };
     }
-    else if (this->flags.has(MessageFlag::RedeemedHighlight) ||
-             this->flags.has(MessageFlag::RedeemedChannelPointReward))
+
+    if (this->flags.has(MessageFlag::RedeemedHighlight) ||
+        this->flags.has(MessageFlag::RedeemedChannelPointReward))
     {
-        return SBHighlight(
+        return {
             ColorProvider::instance().color(ColorType::RedeemedHighlight),
-            SBHighlight::Default, true);
+            ScrollbarHighlight::Default,
+            true,
+        };
     }
-    else if (this->flags.has(MessageFlag::ElevatedMessage))
+
+    if (this->flags.has(MessageFlag::ElevatedMessage))
     {
-        return SBHighlight(ColorProvider::instance().color(
-                               ColorType::ElevatedMessageHighlight),
-                           SBHighlight::Default, false, false, true);
+        return {
+            ColorProvider::instance().color(
+                ColorType::ElevatedMessageHighlight),
+            ScrollbarHighlight::Default,
+            false,
+            false,
+            true,
+        };
     }
-    else if (this->flags.has(MessageFlag::FirstMessage))
+
+    if (this->flags.has(MessageFlag::FirstMessage))
     {
-        return SBHighlight(
+        return {
             ColorProvider::instance().color(ColorType::FirstMessageHighlight),
-            SBHighlight::Default, false, true);
+            ScrollbarHighlight::Default,
+            false,
+            true,
+        };
```

Comment:
Not from this PR: The `ScrollbarHighlight` needs some refactoring. The arguments `isRedeemedHighlight`,
`isFirstMessageHighlight`, and `isElevatedMessageHighlight` are mutually exclusive - shouldn't this be refactored to an enum? Even if they're not mutually exclusive, a bit field would be more readable.

Reply:
Definitely should be enums yeah